packages/gatsby-worker/src/child.ts

```diff
@@ -102,6 +103,8 @@ if (process.send && process.env.GATSBY_WORKER_MODULE_PATH) {
   }
 
   process.on(`message`, messageHandler)
+
+  ensuredSendToMain([WORKER_READY])
```

Comment:
We didn't have init messaging and seems like in some cases initial messages (before child set `on('message')` handler) are lost. The change here is that child sends "ready" message once it's initialized and workerpool waits for it before sending task to process